### ISSUE TITLE
Fix dependencies

### ISSUE THREAD

**Issue by ruscon** [overnested/nestjs-octokit#7]
1. Need to move this to the `peerDependencies`
```
  "dependencies": {
    "@nestjs/common": "^8.2.5",
    "@nestjs/core": "^8.2.5",
    "octokit": "^1.7.1",
    "reflect-metadata": "^0.1.13"
  }
```
2. Allow `@nestjs/*` all versions like this
```
  "peerDependencies": {
    "@nestjs/common": "^8.2.5 || ^9.0.0 || ^10.0.0",
    "@nestjs/core": "^8.2.5 || ^9.0.0 || ^10.0.0",
  }
```
3. Need to upgrade all other packages

**Comment by codiophile** [comment]
This project looks useful, but it seems abandoned. How did you end up solving this issue? Did you fork it, write your own or are you perhaps using `octokit` directly?

**Comment by ruscon** [comment]
@codiophile I created my own copy of the module

**Comment by Helveg** [comment]
I don't think it is abandoned, @alitnk is the maintainer, are you still on this project? If not I volunteer as maintainer in your absence :)

**Comment by cerpinn** [comment]
I'm not. I'll archive the repository. Feel free to fork and maintain!
